test/test.js

```diff
@@ -1,9 +1,13 @@
-const assert = require('assert');
+
 const got = require('got');
-const server = require('../src/index.js');
+const server = require('../src');
+const defaultValues = require('../defaultValues.js');
+const chai  = require('chai');
+
+const expect = chai.expect;
 
 describe('return type check', () => {
-    const newInstance = new server.MyServer(8000);
+    const newInstance = new server.MyServer(defaultValues.port);
```

Comment:
You shouldn't pass default values explicitly in tests, since they must be *default* and applied automatically.